### PR TITLE
(feat): add I say event to allure

### LINKED ISSUE #2104: I.say() does not join the Allure report.
#### What are you trying to achieve?
Get the `I.say` step in Allure report.
#### What do you get instead?
Miss `I.say` step inside Allure report.
> Console output with `npx codeceptjs run --plugins allure --steps`:

```bash
I click "[data-test-selector=submit-
Вижу ошибку 
I wait for element "[data-test-selector=username-error]", 4
```

> Test source code:

```js
# auth_test.js
Feature('Auth');
Scenario('Demo.', async (I) => {
    I.click(loginPage.submitButton);
    I.say('Вижу ошибку логина!');
    I.waitForElement(loginPage.error.login, 4);
});
```
> Allure report output:

Screenshot from 2019-12-24 15-25-07

### Details

* CodeceptJS version: v2.3.3
* NodeJS Version: v13.3.0
* Operating System: Ubuntu, Linux 4.15.0-72-generic x86_64
* puppeteer: v1.20.0
* Configuration file: 

```js
  plugins: {
    allure: {
      enabled: true,
      outputDir: './output/allure-results'
    }
}
```

### PR DESCRIPTION
**Pull request by PeterNgTr**
## Motivation/Description of the PR
Resolves #2104 

I think the timestamps could be left as 0s or any suggestion on this would be nice

Screenshot 2020-01-17 at 14 15 57

## Type of change

- [ ] Breaking changes
- [x] New functionality
- [ ] Bug fix
- [ ] Documentation changes/updates
- [ ] Hot fix
- [ ] Markdown files fix - not related to source code

### CODE REVIEW

**Comment by storenth** [comment]
@Vorobeyko @PeterNgTr I can review and retest if you grant write access.

**Comment by elukoyanov** [comment]
@PeterNgTr is this problem for allure only?

Or there are problem, which fix all other helpers

File: lib/plugin/allure.js [hunk 1]
```diff
@@ -244,6 +244,13 @@ module.exports = (config) => {
     }
   });
 
+  event.dispatcher.on(event.step.comment, (step) => {
+    reporter.startStep(`I say "${step.toString()}"`);
```
Comment (DavertMik) [inline lib/plugin/allure.js]:
```suggestion
    reporter.startStep(step.toString());
```

No need for `I.say` here, we need only the text.

**Comment by PeterNgTr** [comment]
> @PeterNgTr is this problem for allure only?
> 
> Or there are problem, which fix all other helpers

I think whatever helper is used would be fine with adding this in allure plugin directly.

**Comment by DavertMik** [comment]
> I can review and retest if you grant write access.

Please do more reviews and become part of our core team to get write access.

Meanwhile, you can always test changes from fork's banrch: `PeterNgTr:feat-add-say-event-to-allure`

**Pull request merged**